### DISCUSSION TITLE
Chromatic Tuner

### DISCUSSION THREAD

**Discussion by subderground** [DISTRHO/Cardinal#566] [Q&A]
Hi,
I was looking for an audio chromatic tuner in Cardinal. But it seems that's not available now (Maybe I miss it).
I search and find one ; "Hot Tuna" by Nysthi (but freeware).
![HotTuna](https://github.com/DISTRHO/Cardinal/assets/105162700/57dcde1d-82a2-4d1d-a2f5-e04eb77279f7)
If I reverse the name -> Ihtsyn, I find open source modules under this name Ihtsyn (some are available in Cardinal.. but not "Hot Tuna" or "Anut Toh" ;-) ).
Is it the same module developer ? Could we ask for a fork of Hot Tuna to add to cardinal ? And globally, did Cardinal reach its maximum capacity of modules ? Or soon ?

Thanks

**Comment by dromer** [comment]
You cannot add closed source modules and Nysthi is closed. You can't "fork" closed source since you don't have .. the source.

The Ihtsyn modules is a small selection based on GPL code that was split from Nysthi after requesting their source (as required by the GPL).

**Comment by subderground** [comment]
Ok thanks for the infos. I think there is no Audio Chromatic Tuner from another developer available as "rack module". Perhaps in the futur...
Have a nice day :-)
